installer/Migration/V5_1_0/lang-string/recruitment.yaml

```
@@ -131,3 +131,28 @@ langStrings:
   -
     value: 'Candidate History'
     unitId: candidate_profile
+
+  - value: 'Add Vacancy'
+    unitId: add_vacancy
+
+  - value: 'Edit Vacancy'
+    unitId: edit_vacancy
+
+  - value: 'Vacancy Name'
+    unitId: vacancy_name
+
+  - value: 'Description'
+    unitId: escription
```

```suggestion
    unitId: description
```
also, this string is there in `general` group. 
@SharukaNP we need unit test for duplicate check

---

removed